projects/burnr/src/hooks/api/useApiCreate.ts

```diff
@@ -1,17 +1,15 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 /* eslint-disable @typescript-eslint/no-unsafe-call */
-/* eslint-disable @typescript-eslint/no-unsafe-member-access */
-
 import { useEffect, useState } from "react"
 import { ApiPromise } from "@polkadot/api"
 import { logger } from "@polkadot/util"
 import { ScProvider } from "@substrate/connect"
 import { ALL_PROVIDERS, BURNR_WALLET } from "../../utils/constants"
-import { useIsMountedRef } from ".."
+import { useIsMountedRef } from "./useIsMountedRef"
 
 const l = logger(BURNR_WALLET)
 
-export default function useApiCreate(): ApiPromise {
```

Comment:
Why is the `default` not correct?

Reply:
It is not a matter of `not good`. Its a matter of personal decision and consistency across code-base. 
TBH I prefer naming exports instead of default as it makes less error-prone and more consistent. But this is a personal opinion ofc. Both ways (using default or naming exports) are correct